app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,82 @@
+class ExpensesController < ApplicationController
+  before_action :set_user
+  before_action :set_category
+  before_action :set_expense, only: %i[show edit update destroy]
+
+  # GET /expenses or /expenses.json
+  def index
+    @expenses = @category.expenses
+  end
+
+  # GET /expenses/1 or /expenses/1.json
+  def show; end
```

Comment:
[ OPTIONAL]
You can consider remvoving actions not used from your controllers. This makes your codebase clean and professional.